sass/directives/_headers.scss

```diff
@@ -129,34 +104,36 @@
     p {
       max-width: 28em;
       margin-bottom: 0;
-      font-size: $hero-text-size;
+      font-size: strip-units($hero-text-size) * 1rem;
```

Comment:
it's okay to leave this unchanged since i have a typography story coming up, but here's why I wouldn't make this change here:

It looks like you're using `strip-units` to get to rems. I'd change the variable value in `sass/variables/_sizing.scss` instead. Anything using `hero-text-size` should be using the same value, and by making the change in a specific place rather than on the variable that multiple things use, this could end up having a different font size from anything else when the font size should be the same.

Also, the reason we use variables is so that we can make the change on the variable and not have to update anything else. When I go to update to the new typography, I'll want to change the value of $hero-text-size and not have to search for other instances that might have been altered.

We can talk through this more if you want!